query-planner/src/macros.rs

```diff
@@ -1,4 +1,3 @@
-#[macro_export]
```

Comment:
No issue with this (as I don't know any better), just would like to understand why we don't need it anymore.

Reply:
IIUC it makes the macro public outside of the crate. I don't think we ever needed it. Either I misunderstood what it does, or maybe we had integration tests that use these macros? (integration tests are "external" to the crate)